.github/workflows/drupal_rector.yaml

```diff
@@ -0,0 +1,44 @@
+name: rector_examples
+
+on:
+    pull_request: null
+    push:
+        branches:
+            - master
+
+jobs:
+    rector_examples:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v2
+            -   uses: shivammathur/setup-php@v2
+                with:
+                    php-version: 7.3
+                    coverage: none # disable xdebug, pcov
+                    extensions: "intl"
+
+            # Run standalone install in non-root package, ref https://github.com/rectorphp/rector/issues/732
+            -   run: |
+                    # Install latest Drupal
+                    composer create-project drupal/recommended-project drupal --dev --no-progress
+
+            -   run: |
+                    # Install drupal-rector (based on rector 0.7.x)
+                    cd drupal
+                    composer require palantirnet/drupal-rector:dev-rector-0.7 --no-progress
```

Comment:
Does this need to be updated to remove the branch name?

Do you want to do that after merging?

Reply:
Yes, after merging we would change this